src/nidmm/examples/nidmm_fetch_waveform.py

```diff
@@ -21,7 +21,7 @@ def example(resource_name, options, function, range, points, rate):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a waveform acquisition using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digital Multimeter module.')
```

Comment:
remove "module". If anything we should say "instrument" per documentation guidelines but in this case it's not needed.

Reply:
Update